server.js

```
@@ -0,0 +1,221 @@
+                    <html lang="en">
+                    
+                    <head>
+                        <meta charset="UTF-8">
+                        <title>The Elements</title>
+                        <link rel="stylesheet" href="/styles.css">
+                    </head>
+                    
+                    <body>
+                        <h1>The Elements</h1>
+                        <h2>These are all the known elements.</h2>
+                        <h3>These are ${newFiles.length} of them.</h3>
+                        <ol>`;
+                    for (let i = 0; i < newFiles.length; i++) {
+                        let newList = `<li>
+                        <a href="/${newFiles[i]}">${newFiles[i].split('.')[0]}</a>
+                    </li>`;
+                        newIndex += newList;
+                    };
+                    let htmlEnd = `    </ol>
+                    </body>
+                    
+                    </html>`;
+                    fs.writeFile('./public/index.html', newIndex + htmlEnd, (err) => {
+                        if (err) {
+                            throw err;
+                        }
+                        console.log('Index.html updated!');
+                    });
+                });
+                res.end();
+            };
+        }
+        // DELETE METHOD
+        if (req.method === 'DELETE') {
+            fs.readdir(`./public/`, (err, files) => {
+                if (files.includes(URL.slice(1)) === false) {
+                    res.writeHead(500, {
+                        'Date': `${date}`,
+                        'Content-Type': `application/json`,
+                    });
+                    res.write(`{ "error" : "resource ${URL} does not exist" }`)
+                    res.end();
+                }
+                fs.unlink(`./public${URL}`, (err) => {
+                    if (err) {
+                        return console.log(err);
```

This should return an error to the client using either a 400/500 error